.circleci/config.yml

```diff
@@ -109,15 +110,19 @@ jobs:
       - run:
           name: Run pytest
           command: |
-            source env/bin/activate
-            mkdir test-results
-            pytest --junitxml=test-results/junit.xml
+            source flask/env/bin/activate
+            mkdir flask/test-results
+            pytest --cov --cov-report=html --junitxml=flask/test-results/junit.xml flask
       - store_test_results:
-          path: test-results
+          path: flask/test-results
+      - store_artifacts:
```

Comment:
u are storing the artifacts twice?

Reply:
What do you mean exactly? As previously, the `test-results/` directory is stored via `store_test_results` and `store_artifacts`. Now additionally `flask/htmlcov` is stored (but not required anymore when the codecov integration works).